.github/workflows/tests.yml

```diff
@@ -0,0 +1,64 @@
+name: Run tests
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
+      DOCKER_COMPOSE: docker-compose -f docker-compose.yml -f docker-compose.frontend.yml -p circles
```

Comment:
I saw this `docker-compose.frontend.yml` file also in `circles-core`, but it doesn't exist anywhere, right?

Reply:
Yeah you are right, nice catch!